### ISSUE TITLE
need to show genes for variants

### ISSUE THREAD

**Issue by nlwashington** [monarch-initiative/monarch-legacy#800]
there is no listing for the affected gene(s) of a given variant.

for example:
http://beta.monarchinitiative.org/variant/MGI:4868458

should show the various genes it affects:
http://www.informatics.jax.org/marker/MGI:4868458

if we don't know what all the affected genes are at data ingest time, then we won't have them in the graph.  this could happen if our sources have incomplete data, but also due to assembly differences between sources.

we can solve this a couple ways:
1.  materialize new links between variants and their affected genes as a post-processing step during the graph load/processing.
2.  we could add some new cypher downstream when generating associations between a variant and it's affected gene(s)...such that if a variant lies within or spans a gene, then it should have an overlaps (RO:0002131) relationship.  we can use any coordinates found. 

the first has the advantage of generating complete data within the graph, making subsequent cypher calls follow just a single pattern.  but this could be an expensive step.  the latter is faster to implement and process and may be good enough.  

comments @ccondit or @cmungall ?

**Comment by ccondit** [comment]
Graph pattern looks like this:
![image](https://cloud.githubusercontent.com/assets/2066940/8583216/bd1d6042-2580-11e5-97e0-7ad865953106.png)
